### PR TITLE
Revert "network annotation to hold egress NAT gateway IP"

### PR DESCRIPTION
**Pull request by liuyuan10**
This reverts commit 79f5d136c746eeff2f92df8f0d315c6ecb70c35e.

This annotation is not part of network api

### CODE REVIEW

File: apis/network/v1/annotations.go [hunk 1]
```diff
@@ -63,12 +63,10 @@ const (
 	AutoGenAnnotationValTrue = "true"
 	// NorthInterfacesAnnotationKey is the annotation key used to hold interfaces data per node.
 	NorthInterfacesAnnotationKey = "networking.gke.io/north-interfaces"
-	// NICInfoAnnotationKey specifies the mapping between the fist IP address and the PCI BDF number on the node.
+	// NICInfoAnnotationKey specifies the mapping between the fist IP addresse and the PCI BDF number on the node.
```
Comment (bowei) [inline apis/network/v1/annotations.go:66]: Typo

Comment (liuyuan10) [reply apis/network/v1/annotations.go]: done

**Comment by yswe** [comment]
@liuyuan10 , could you update the CHANGELOG.md file please?

**Comment by liuyuan10** [comment]
> @liuyuan10 , could you update the CHANGELOG.md file please?

done. removed it

**Review by shouri007 — APPROVED** [review]

**Pull request merged**